src/ui/public/timepicker/absolute_panel/kbn_timepicker_absolute_panel.html

```diff
@@ -12,7 +15,10 @@
             ng-click="setToNow({key:'from'})"
             kbn-accessible-click
           >
-            Set To Now
+            <FormattedMessage
+              i18n-id="common.ui.timepicker.setToNow"
```

Comment:
I wonder if it would be better to have a consistent prefix? In the previous file, it was: `kbn.advancedSettings.timepicker` but in this one, it's `common.ui.timepicker`.

Reply:
I tend to agree, but each area in the code has its own prefix, enforced by the i18n validator